lib/note-editor/index.tsx

```diff
@@ -124,11 +124,12 @@ export class NoteEditor extends Component<Props> {
 
     const isTrashed = !!note.deleted;
 
+    const lastUpdatedDate =
+      (lastUpdated && new Date(lastUpdated).toLocaleString()) || 'Unknown';
```

Comment:
I'm sure you'll see this in hindsight, but this is structurally the ternary operator…

```ts
const lastUpdatedDate = lastUpdated
	? new Date(lastUpdated).toLocaleString()
	: 'Unknown';
```

Reply:
hahahahahahahahah, brain not working.

Reply:
Fixed